packages/core/content-releases/admin/src/components/CMReleasesContainer.tsx

```diff
@@ -250,15 +251,16 @@ export const CMReleasesContainer = () => {
   const { formatMessage } = useIntl();
   const {
     isCreatingEntry,
-    allLayoutData: { contentType },
+    hasDraftAndPublish,
+    initialData: { id: entryId },
+    slug,
```

Comment:
@markkaylor @simotae14 I would like your opinion on this 🤔 

slug looks like a great way to get always the uid of both collectionTypes and singleTypes, but for some reason the type of slug is just string (not ContentType.UID), so I don't know

Reply:
Sounds good to me. Did you look how slug was typed inside the useCMEditViewDataManager() hook? Maybe its inferred as a string?

Reply:
It's typed as a string, that made me doubt, but looks like is right to use it 🤔

Reply:
it's a string, because it comes from the router, you can't really narrow it to that particular type of string.